docs/sql-programming-guide.md

```diff
@@ -1587,6 +1580,10 @@ options.
       Note that this is different from the Hive behavior.
     - As a result, `DROP TABLE` statements on those tables will not remove the data.
 
+ - From Spark 2.0.1, `spark.sql.parquet.cacheMetadata` is no longer used. See
+   [SPARK-16321](https://issues.apache.org/jira/browse/SPARK-16321) and
+   [SPARK-15639](https://issues.apache.org/jira/browse/SPARK-15639) for details.
```

Comment:
These two jiras are wrong.

Reply:
https://github.com/apache/spark/pull/13701 is `[SPARK-15639][SPARK-16321][SQL] Push down filter at RowGroups level for parquet reader`.

It's removed here.
- https://github.com/apache/spark/pull/13701/files#diff-ee26d4c4be21e92e92a02e9f16dbc285L625

Reply:
There is no caller for `initializeLocalJobFunc `. Thus, `initializeLocalJobFunc ` is a dead code.

Reply:
Oh, then, it's another transitive search.

Reply:
It sounds like https://issues.apache.org/jira/browse/SPARK-13664 is the one that removes the usage of this conf.

Reply:
For `initializeLocalJobFunc`, I think @HyukjinKwon 's comment is more correct.

678b96e77b [SPARK-14535][SQL] Remove buildInternalScan from FileFormat

Reply:
Maybe, here.
- https://github.com/apache/spark/commit/678b96e77bf77a64b8df14b19db5a3bb18febfe3#diff-51313f5011e9f5a41af166d766950ba2L400

Reply:
`buildInternalScan` is dead since SPARK-13664?

Reply:
I will update like this.
```
 - From Spark 2.0.0, `spark.sql.parquet.cacheMetadata` is no longer used.
   See [SPARK-13664](https://issues.apache.org/jira/browse/SPARK-13664) for details.
```

Reply:
Thank you!

Reply:
Hi, I'm new to spark.  I wonder how to disable metadata caching after deleting this conf. I created an external table, and the parquet files in specified location are updated daily, So I want to disable metadata caching rather than executing 'refresh table xxx'.

Reply:
Hi, @zzl1787 . This is Apache Spark 2.3. In Apache Spark 2.3, the metadata cache is not controlled by this parameter.

Reply:
@dongjoon-hyun Ok, got this, and thank you. Finally I find the parameter to control this.
`spark.sql.filesourceTableRelationCacheSize = 0`
This will disable the metadata cache.